developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -230,6 +231,8 @@ enum LinePrefixType { lptNone, lptKeymanAndKeymanWeb, lptKeymanWebOnly, lptKeyma
 /* Compile target */
 
 kmcmp_CompilerMessageProc msgproc = NULL;
+kmcmp_LoadFileProc loadfileproc = NULL;
```

Comment:
Was hoping to find the declaration for `loadfileproc`, so I could refer to documentation and properly infer what this method actually _does_ at this point... but this was the closest thing I could find... and it's not particularly helpful.

Reply:
The declaration was in a previous PR. Here 'tis:

https://github.com/keymanapp/keyman/blob/c95e232f3c51073b292c4706a84c14be9898da6b/developer/src/kmcmplib/include/kmcmplibapi.h#L51